src/php/utils/docker/docker-php-dev-mode

```diff
@@ -29,7 +29,7 @@ case "$1" in
 			exit 1
 		fi
 
-		pecl install xdebug-stable
+		pecl install xdebug-stable || pecl install xdebug-2.9.8
```

Comment:
Since we'll need different versions depending on the PHP version, should we make the version as argument of this script?

Reply:
Yes, that would be the better long-term solution. Another thing I'm looking at and already did in my own images to use a different extension installer. Which has some advantages, including dropping nearly 50% off the size of my images.

Reply:
sounds good 👍🏼